quipucords/tests/quipucords/test_user.py

```diff
@@ -54,3 +60,82 @@ def test_reject_user_attribute_similarities(self):
             ValidationError, match="The password is too similar to the username"
         ):
             validate_password("test", user=test_user)
+
+
+@pytest.mark.django_db
+class TestCreateOrUpdateUser:
+    """Test create_or_update_user with various inputs."""
+
+    def test_new_user_no_password(self, faker):
+        """Test create_or_update_user with a new username and no password."""
+        username = faker.name()
+        email = faker.email()
+        password = None
+        created, updated, generated_password = create_or_update_user(
+            username, email, password
+        )
+        assert created
+        assert not updated
+        assert generated_password
+        assert User.objects.filter(username=username).exists()
+        with override_settings(AXES_ENABLED=False):
+            assert Client().login(username=username, password=generated_password)
```

Comment:
Is there a reason to not use [`client` fixture](https://pytest-django.readthedocs.io/en/latest/helpers.html#client-django-test-client) here?

Reply:
Same for other tests using `django.test.Client` directly

Reply:
Oh, I might have forgotten about that one! Trying it now…

Reply:
Good catch! It looks like that works perfectly with no other changes. I have amended and force pushed the last two commits to use the `client` fixture. 👍